packages/suite/src/views/dashboard/AssetsView/AssetCard/AssetCard.tsx

```diff
@@ -34,7 +48,12 @@ const StyledCard = styled(Card)`
 `;
 
 const Content = styled.div`
```

Comment:
Can be done with Column. The padding is not needed, I think.

Reply:
https://github.com/trezor/trezor-suite/pull/14996/commits/e57c6a41d63a7760b80706c1b73811aeca389776